data/json/recipes/food/dairy_products.json

```
@@ -37,5 +37,23 @@
     "autolearn": true,
     "qualities": [ { "id": "COOK", "level": 1 } ],
     "components": [ [ [ "milk_cream_jar", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "ghee",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DAIRY",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "charges": 1,
+    "time": "20 m",
```

```suggestion
    "time": "30 m",
```
Every recipe I find estimates 30 mins

---

You're right, that was an oversight. Thank you!